### PR TITLE
Useless command if AddModule spryk is used

### PR DESCRIPTION
**Pull request by amansilla**
After starting to play with the SDK I was in this case mostly interested in testing how it can be used for generating basic module code so development performance can be improved.  

After running the command `spryker-sdk generate:php:add-module --module=VisaPayment --namespace=Pyz -vvv` the only result is that in best case one folder is created, nothing awesome I would say.

If instead of using the `AddModule`, `AddModuleDefault` is used, real boilerplate code is being generated e.g. Shared and Zed module directories, transfer objects definition file, Zed Config and DependencyProvider, Facade. Entity Manager and repository.

With this change, the command can add some value to our Spryker developers during development.

### CODE REVIEW

**Review by pavelmaksimov25 — APPROVED** [review]

File: src/Extension/Resources/config/task/SprykAddModuleTask.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ id: "generate:php:add-module"
 short_description: "Generate a new Spryker module in the current project"
 help: ~
 stage: build
-command: "php %sdk_dir%/vendor/bin/spryk-run AddModule %organization% %coreLevel% %module%"
+command: "php %sdk_dir%/vendor/bin/spryk-run AddModuleDefault %organization% %coreLevel% %module%"
```
Comment (pavelmaksimov25) [inline src/Extension/Resources/config/task/SprykAddModuleTask.yaml:6]: Task version must be updated in order to update it in the existing initialized sdk

**Review by pavelmaksimov25 — CHANGES_REQUESTED** [review]

File: src/Extension/Resources/config/task/SprykAddModuleTask.yaml (hunk 1 above)
Comment (amansilla) [reply src/Extension/Resources/config/task/SprykAddModuleTask.yaml:6]: Ups I forgot about that. I will update it.

Comment (amansilla) [reply src/Extension/Resources/config/task/SprykAddModuleTask.yaml:6]: @pavelmaksimov25 task version has been updated. ;)

**Comment by amansilla** [comment]
@pavelmaksimov25 I have updated the version to `1.1.2`. I hope it's fine.

**Review by pavelmaksimov25 — APPROVED** [review]

**Pull request merged**